clients/sirene-insee/siret.ts

```diff
@@ -221,16 +238,19 @@ export const mapEtablissementToDomainObject = (
       trancheEffectifsEtablissement,
       anneeEffectifsEtablissement
     ),
-    adresse: formatAdresse(
-      adresseEtablissement.complementAdresseEtablissement,
-      adresseEtablissement.numeroVoieEtablissement,
-      adresseEtablissement.indiceRepetitionEtablissement,
-      adresseEtablissement.typeVoieEtablissement,
-      adresseEtablissement.libelleVoieEtablissement,
-      adresseEtablissement.codePostalEtablissement,
-      adresseEtablissement.libelleCommuneEtablissement,
-      adresseEtablissement.libellePaysEtrangerEtablissement
-    ),
+    adresse: formatAdresse({
+      complement: complementAdresseEtablissement,
+      numeroVoie: numeroVoieEtablissement,
+      indiceRepetition: indiceRepetitionEtablissement,
+      typeVoie: typeVoieEtablissement,
+      libelleVoie: libelleVoieEtablissement,
+      codePostal: codePostalEtablissement,
+      libelleCommune: libelleCommuneEtablissement,
+      distributionSpeciale: distributionSpecialeEtablissement,
+      codeCedex: codeCedexEtablissement,
+      libelleCommuneCedex: libelleCedexEtablissement,
+      pays: libellePaysEtrangerEtablissement,
```

Comment:
I did not use `pays` or the other country fields. Should I add them ?

Reply:
@XavierJp

Reply:
yes !